spec/models/user_spec.rb

```diff
@@ -689,7 +689,7 @@
   end
 
   describe 'user suspension' do
-    let(:user) { User.new }
+    let(:user) { create(:user, email: 'foo@gmail.com') }
```

Comment:
do we need the `@gmail.com` here? We still create suspension records even for other domains right?

Reply:
Right! we don't need @gmail.com